### PR TITLE
APPS-313 Add summary of changes between v1 and v2

### PR DESCRIPTION
**Pull request by commandlinegirl**

### CODE REVIEW

**Review by jdidion — DISMISSED** [review]
LGTM thanks!

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
 # Release Notes
 
+
+## 2.0.0 Release Notes Summary
+
+A summary of changes in the 2.0.0 version of dxCompiler comparing to the previous dxWDL version 1:
```
Comment (emiloslavsky) [inline RELEASE_NOTES.md:6]: Is this compared against the latest version of dxWDL (v1.48.2) ?

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+
+User-facing changes:
+* Changed the name and all references from dxWDL to dxCompiler, dxCompiler-\*.jar is now used to compile an application or a workflow
+* Replaced Cromwell WOM (parser, type checker and evaluator) with [wdlTools](https://github.com/dnanexus-rnd/wdlTools), a library maintained by DNAnexus
+* Added support for per-workflow and per-scatter chunk size settings to extras.json
+* Added `streamFiles` compile option
+* Updated the mechanism of comparing instance types to select the cheapest one for execution
+* Improved file resolving and caching
+* Optimized bulk description of files by replacing `system/describeDataObjects` with `system/findDataObjects` API call and scoping file search to projects
+* Made array type-checking less strict to coerce between empty and non-empty array types
```
Comment (emiloslavsky) [inline RELEASE_NOTES.md:16]: less strict than Cromwell?   I though Cromwell was more relaxed.

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+* Made it possible to run apps built using dxCompiler (after applets to apps conversion using dx-toolkit SDK, min. v0.296)
```
Comment (emiloslavsky) [inline RELEASE_NOTES.md:17]: what's that about?

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,39 @@
...
+* Increased file name disambiguation directory size limit to 5000
+* Increased number of API retries to 10
```
Comment (emiloslavsky) [inline RELEASE_NOTES.md:19]: during compilation of WDL?   Or during the execution of DNAnexus apps generated with dxCompiler?

**Review by emiloslavsky — CHANGES_REQUESTED** [review]

File: RELEASE_NOTES.md (hunk 1 above)
Comment (commandlinegirl) [reply RELEASE_NOTES.md:6]: It is compared against v1.47.2, i.e. the version after which the first 2.0.0-rc was released, not against the latest dxWDL which was v1.50 (Dec 9th).

I compared against v1.47.2 to include **all** the changes that went into the `2.0.0-rc*` versions.

File: RELEASE_NOTES.md (hunk 3 above)
Comment (commandlinegirl) [reply RELEASE_NOTES.md:17]: It's about the ability to run apps, not only applets, compiled from WDL.

File: RELEASE_NOTES.md (hunk 4 above)
Comment (commandlinegirl) [reply RELEASE_NOTES.md:19]: During the compilation and execution (it's a retry count for any http API request to DX api). Will clarify.

File: RELEASE_NOTES.md (hunk 2 above)
Comment (commandlinegirl) [reply RELEASE_NOTES.md:16]: It's actually "less strict than in the previous version of wdlTools" so it looks like this doesn't fit here. Removed

**Pull request merged**